charts/node-analyzer/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: node-analyzer
 description: Sysdig Node Analyzer
 # currently matching Sysdig's appVersion 1.14.34
-version: 1.35.11
+version: 1.35.12
```

```suggestion
version: 1.36.0
```
For consistency with the version bump in the kspm ananlyzer